packages/server/src/api/controllers/debug.ts

```diff
@@ -0,0 +1,45 @@
+import os from "os"
+import process from "process"
+import { env } from "@budibase/backend-core"
+
+export async function systemDebugInfo(ctx: any) {
```

Comment:
This can be updated to make use of response body types. 
```ts
ctx: UserCtx<void, GetDiagnosticsResponse>
```
// budibase/types -> src/api
```ts
export interface GetDiagnosticsResponse {
   // as appropriate
}
```